assets/controllers.json

```diff
@@ -0,0 +1,25 @@
+{
+    "controllers": {
+        "@symfony/ux-autocomplete": {
+            "autocomplete": {
+                "enabled": true,
+                "fetch": "eager",
+                "autoimport": {
+                    "tom-select/dist/css/tom-select.default.css": true,
+                    "tom-select/dist/css/tom-select.bootstrap4.css": false,
+                    "tom-select/dist/css/tom-select.bootstrap5.css": false
+                }
+            }
+        },
+        "@symfony/ux-live-component": {
+            "live": {
+                "enabled": true,
+                "fetch": "eager",
+                "autoimport": {
+                    "@symfony/ux-live-component/dist/live.min.css": true
+                }
+            }
+        }
+    },
```

Comment:
It seems `@symfony/ux-live-component` is no more loaded according to shop and admin bundle `controller.json` files:
https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/AdminBundle/Resources/assets/controllers.json
https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/ShopBundle/Resources/assets/controllers.json

Reply:
Not in controller.json, but in app.js, in a custom way. Don't know why it wasn't used in controllers.json though

https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/ShopBundle/Resources/assets/app.js#L11

https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/AdminBundle/Resources/assets/app.js#L11